src/persons/PersonForm.js

```diff
@@ -87,7 +87,7 @@ let PersonForm = (props: Props) => {
                     <label>
                       <Trans>Tipus de document de identitat</Trans>
                     </label>
-                    <Field data-test="document_identitat" name="tipus_document_identitat" component={Select} fullWidth>
+                    <Field data-test="identitat" name="identitat" fullWidth component={Select} >
```

Comment:
Can't see how this name improves comprehension.

Reply:
Was a mistake. fixed